### ISSUE TITLE
Support on Multi-GPU？

### ISSUE THREAD

**Issue by Minotaur-CN** [dbolya/yolact#3]
Hi, dbolya,

I did not find dataparallel in your yolact.py, which define the model. So the code in your repo did not support multi-gpu properly?
I tried simple CUDA_VISIBLE_DEVICES to assign multi-gpu, but the performance is not right according to the train log.

Thanks!

**Comment by chongzhou96** [comment]
The repo supports multi-gpu, and Dataparallel is used in train.py.
[https://github.com/dbolya/yolact/blob/master/train.py#L174](url)
Can you provide more details about what seems to be not right? 

Thanks.

**Comment by dbolya** [comment]
I would add that to train with multiple GPUs, you should turn off JIT (the way I implemented it doesn't play well with multiple GPUs): add `--no_jit` to the commandline arguments.

Also, if your batch size is less than 8 per GPU, you should also freeze batch norm by adding `'freeze_bn': True` to the config you use in `data/config.py` (e.g., ctrl+f `yolact_base_config` and add it to the dict there).

**Comment by anustupdas** [comment]
Hi,

So what are the changes one needs to make to make it train on multiple GPUs?
Any lead will be highly appreciated. 

Thanks. :)

**Comment by dbolya** [comment]
See #8 for all of the changes. One of these days I'll add the necessary changes as a command-line argument.

**Comment by dbolya** [comment]
For future reference: I removed `--no_jit` and now it'll just turn JIT off if it detects more than one GPU.
